.github/workflows/t3000-demo-tests-impl.yaml

```diff
@@ -23,7 +28,7 @@ jobs:
       ARCH_NAME: ${{ matrix.test-group.arch }}
       LOGURU_LEVEL: INFO
       LD_LIBRARY_PATH: ${{ github.workspace }}/build/lib
-    runs-on: ["arch-wormhole_b0", "config-t3000", "in-service", "pipeline-perf"]
+    runs-on: ["arch-wormhole_b0", "config-t3000", "in-service", "pipeline-functional", ${{ inputs.extra-tag }}]
```

Comment:
shoud set extra-tag as default to in-service.

That way you can use this workflow to triage OOS units as well

Reply:
oh good catch!